common-npm-packages/docker-common-v2/make.js

```diff
@@ -9,5 +9,6 @@ util.run(path.join(__dirname, 'node_modules/.bin/tsc') + ' --outDir ' + buildPat
 util.cp(path.join(__dirname, 'package.json'), buildPath);
 util.cp(path.join(__dirname, 'package-lock.json'), buildPath);
 util.cp(path.join(__dirname, 'module.json'), buildPath);
+util.cp('-r', 'tools', buildPath);
```

Comment:
Can you please educate me about this change ?

Reply:
This was a mistake on my part. This line was needed for testing purposes, and I thought I reverted it back to the original version of the file, but it looks like I didn't. I have gone ahead and removed this line from the file.